devops/jobs/CreateSandbox.groovy

```diff
@@ -192,6 +192,9 @@ class CreateSandbox {
                 booleanParam("learner_portal_enterprise",false,"Enable Enterprise Learner Portal MFE")
                 stringParam("learner_portal_enterprise_version","master","The repository version of the frontend-app-learner-portal-enterprise")
 
+                booleanParam("ora-grading",false,"Enable ORA Grading MFE")
+                stringParam("ora-grading_version","master","The repository version of the frontend-app-ora-grading")
```

Comment:
This sets up `ora-grading_version` as an environment variable. Hyphens aren't super kosher in that context; I think many implementations would work, but stylistically we really should use only alphnumeric and underscores here.